modules/apps/27-interchain-accounts/keeper/keeper_test.go

```diff
@@ -76,3 +98,33 @@ func (suite *KeeperTestSuite) TestGetPort() {
 	port := suite.chainA.GetSimApp().ICAKeeper.GetPort(suite.chainA.GetContext())
 	suite.Require().Equal(types.PortID, port)
 }
+
+func (suite *KeeperTestSuite) TestGetInterchainAccountAddress() {
+	suite.SetupTest()
+	path := NewICAPath(suite.chainA, suite.chainB)
+	suite.coordinator.SetupConnections(path)
+
+	if err := SetupICAPath(path, "testing"); err != nil {
+		suite.FailNow("failed to configure interchain accounts path")
+	}
```

Comment:
this looks good, but historically we have used

`suite.Require().NoError(err)` which I believe calls `FailNow`. Do you think you could update it? Or do you think `suite.FailNow` is more suitable?

Reply:
Updated! I think either is probably fine, I'm just generally used to using: `if err := funcCall(); err != nil {}` if there is only one return val of type `error`